CMakeLists.txt

```diff
@@ -11,8 +11,14 @@ set(GCF_CLANGFORMAT_STYLE "file" CACHE STRING
     "Parameter pass to clang-format -style=<here>")
 set(GCF_IGNORE_LIST "" CACHE STRING
     "Semi colon separated list of directories to ignore")
+set(GCF_CLANGFORMAT_MINIMAL_VERSION "0.0.0" CACHE STRING
+    "The minimal required version of clang-format")
 set(GCF_SCRIPT ${CMAKE_CURRENT_SOURCE_DIR}/git-cmake-format.py)
 
+if("${CLANG_FORMAT_VERSION}" VERSION_LESS "${GCF_CLANGFORMAT_MINIMAL_VERSION}")
+  message(FATAL_ERROR "clang-format version is ${CLANG_FORMAT_VERSION} less than minimal requirement ${GCF_CLANGFORMAT_MINIMAL_VERSION}")
+endif()
```

Comment:
Instead of that, you can maybe add ${GCF_CLANGFORMAT_MINIMAL_VERSION} into the `find_package(ClangFormat REQUIRED)`.
https://cmake.org/cmake/help/v3.8/command/find_package.html